dist/resources/validationSchema/1.0.0-rc2/IMessageBlock.json

```diff
@@ -113,7 +113,8 @@
           "type": "string"
         },
         "property_value": {
-          "type": "string"
+          "type": "string",
+          "format": "floip-expression"
```

Comment:
Clever !
For the implementation ticket vmo-5090, don't forget the `value` type of IResourceValue interface too.